### ISSUE TITLE
Search input has no corresponding label.

### ISSUE THREAD

**Issue by dbollinger** [cibernox/ember-power-select#1053]
I'm trying to meet WCAG and pass the default rules in the AXE tool used by ember-a11y-test, and I'm stuck with a failure on the search input element when the control is expanded.

In both the `before-options` and `trigger` components, the search input has no corresponding label, making the purpose of the input less clear to assistive technologies.

A configurable `searchLabel` or `ariaSearchLabel` or even `searchLabelledby` property would be minimally intrusive way to solve this problem.

https://dequeuniversity.com/rules/axe/1.1/label

**Comment by mkszepp** [comment]
We have now added the option for label in v8
